Lamina/src/GL/opengl.cpp

```diff
@@ -0,0 +1,80 @@
+#include "../Lamina/include/GL/opengl.hpp"
+
+GLuint lm::GLRenderer::CompileShader(std::string vertexShaderPath, std::string fragmentShaderPath)
+{
+	if (!glfwInit()) return GL_FALSE;
```

Comment:
While I know that this function returns true immediately if glfw is already initialized, it looks a bit weird to have to confirm this numerous times after the application has initialized. If glfw initialization has already failed, the application should have already exited by now. To me, it would be similar to confirming that the OpenGL context was created successfully every time an OpenGL function is called.